client.go

```diff
@@ -409,40 +333,76 @@ func (r *rowWrapper) Write(dest map[string]interface{}) error {
 
 // Columns returns the column names.
 func (r *rowWrapper) Columns() ([]string, error) {
-	columns, err := r.row.Columns()
+	columns, err := r.rows.Columns()
 	if err != nil {
 		return nil, errors.Wrap(err, "makroud: cannot return row columns")
 	}
+
 	return columns, nil
 }
 
 // Scan copies the columns in the current row into the values pointed at by dest.
 // The number of values in dest must be the same as the number of columns in Rows.
 func (r *rowWrapper) Scan(dest ...interface{}) error {
-	err := r.row.Scan(dest...)
+	err := r.scan(dest...)
 	if err != nil {
 		return errors.Wrap(err, "makroud: cannot scan given values")
 	}
 	return nil
 }
 
-// A rowsWrapper wraps a rows from sqlx.
+func (r *rowWrapper) scan(dest ...interface{}) error {
+	// From sqlx source code: Discard sql.RawBytes to avoid weird issues with the SQL driver and memory management.
```

Comment:
```suggestion
	// From https://github.com/jmoiron/sqlx source code: Discard sql.RawBytes to avoid weird issues with the SQL driver and memory management.
```